transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -219,14 +219,27 @@ public ServerBootstrap validate() {
         private final ChannelHandler childHandler;
         private final Entry<ChannelOption<?>, Object>[] childOptions;
         private final Entry<AttributeKey<?>, Object>[] childAttrs;
+        private final Runnable enableAutoReadTask;
 
         ServerBootstrapAcceptor(
-                EventLoopGroup childGroup, ChannelHandler childHandler,
+                final Channel channel, EventLoopGroup childGroup, ChannelHandler childHandler,
                 Entry<ChannelOption<?>, Object>[] childOptions, Entry<AttributeKey<?>, Object>[] childAttrs) {
             this.childGroup = childGroup;
             this.childHandler = childHandler;
             this.childOptions = childOptions;
             this.childAttrs = childAttrs;
+
+            // Task which is scheduled to re-enable auto-read.
+            // It's important to create this Runnable before we try to submit it as otherwise the URLClassLoader may
+            // not be able to load the class because of the file limit it already reached.
```

Comment:
Yap.See [another issue](https://github.com/netty/netty/issues/5962).

What I have done is is..

``` java 

private void initNettyLogger() {
        if(bossGroup != null)
            bossGroup.execute(new Runnable() {
                @Override
                public void run() {
                    throw new RuntimeException("Please ignore this.HSF do this just for init netty logger.");
                }
            });
        if (httpBossGroup != null)
            httpBossGroup.execute(new Runnable() {
                @Override
                public void run() {
                    throw new RuntimeException("Please ignore this.We do this just to init Netty's logger.");
                }
            });
    }

```